README.md

```diff
@@ -15,47 +15,35 @@ Add the plugin and repositories listing to `project.clj`. **NB: You need to add
 :plugins [[s3-wagon-private "1.2.0"]]
 ```
 
-You can store credentials either in an encrypted file or as
-environment variables. For the encrypted file, add this to
-`project.clj`:
+To authenticate to the S3 bucket, you can either:
 
-```clj
-:repositories [["private" {:url "s3p://mybucket/releases/" :creds :gpg}]]
-```
+- Use any of the AWS SDK [chained provider class][chained-provider-class] credential providers. An excerpt of the most commonly used credential providers:
+     - Environment Variables - `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
+     - Java System Properties - aws.accessKeyId and aws.secretKey
```

Comment:
@jebberjeb version "1.3.0-alpha1" supports this. I just tried it out with the following example and it worked for me:

in ~/.aws/credentials:
```
[new-profile]
aws_access_key_id=<access-key-here>
aws_secret_access_key=<secret-key-here>
```
then running: `AWS_PROFILE=new-profile lein deps` with a `project.clj` that has private dependencies configured.

What error are you receiving? Unable to auth to the bucket or unable to load credentials in the provider chain?